.gitignore

```diff
@@ -12,3 +12,6 @@
 *.o
 *.a
 mkmf.log
+
+.DS_Store
+/.idea/
```

Comment:
Not an issue but might be better for you to just add `.idea` directories to your global gitignore. You wouldn't have to set this in each project.